tests/fixtures/fixture_imodc_executable.py

```diff
@@ -0,0 +1,24 @@
+import subprocess
+import sys
+from collections.abc import Callable
+from pathlib import Path
+
+import pytest
+
+from imod_coupler.__main__ import run_coupler
+
+
+@pytest.fixture(scope="session")
+def imodc_executable(imod_coupler_exec_devel: Path) -> Callable[[Path], None]:
+    """Replacement for subprocess.run"""
+    if "pydevd" in sys.modules:
+
```

Comment:
```suggestion
    if "pydevd" in sys.modules:
```

Reply:
Auto-format doesn't like that

Reply:
Interesting. Well, then let's believe in auto-format :)